queue/src/fiskaltrust.Middleware.Localization.QueueDEFAULT/Factories/SignatureItemFactoryDEFAULT.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.Collections.Generic;
+using System.Globalization;
+using fiskaltrust.ifPOS.v1;
+using fiskaltrust.Middleware.Contracts.Factories;
+
+namespace fiskaltrust.Middleware.Localization.QueueDEFAULT.Factories
+{
+    public class SignatureItemFactoryDEFAULT : SignatureItemFactory
+    {
+        public override long CountryBaseState => 000000000;
+
+        public SignatureItemFactoryDEFAULT() { 
+        }
+
+        protected static NumberFormatInfo CurrencyFormatter = new ()
+        {
+            NumberDecimalSeparator = ".",
+            NumberGroupSeparator = "",
+            CurrencyDecimalDigits = 2
+        };
+        
+        public List<SignaturItem> GetSignaturesForPosReceiptTransaction()
+        {
+            var signatures = new List<SignaturItem>
+            {
+                CreateQrCodeSignature("www.fiskaltrust.eu", "<ftCashBoxIdentification>_<ftReceiptIdentification>_<SumOfPayItems>_<responseHashOfThePreviousQueueItem>")
```

Comment:
IMO we will have to replace this with actual values, or is this done in the caller?

Reply:
Need to replace it